src/components/IKContext/props.ts

```diff
@@ -12,17 +9,12 @@ const Props = {
 
 export const IKContextCombinedProps = {
   ...Props,
-  ...IK_IMAGE_PROPS,
-  ...IK_UPLOAD_PROPS,
-  ...IK_VIDEO_PROPS,
+  transformationPosition: PropTypes.oneOf(['path', 'query']),
   ikClient: PropTypes.instanceOf(ImageKit),
```

Comment:
Should be explicit IMO instance of taking props of ImageKit

Reply:
done

Reply:
can variable name be just `IKContextProps`?

Reply:
addressed in [4b16cdb](https://github.com/imagekit-developer/imagekit-react/pull/145/commits/4b16cdbe1163fdf54500da2144bade484797fdcc)